### PR TITLE
fix: pull request button shows PRs now on a contributor card

### LINKED ISSUE #2269: bug: Can't click the pull requests list on the contributor tab
Screen Shot 2023-12-06 at 8 30 04 AM

### PR DESCRIPTION
**Pull request by OgDev-01**
## Description
This PR fixes the issue with the show pull request button in the contributors card. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
fixes #2269 

## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/app/assets/62995161/7ef5fa79-e931-493a-b04a-512f7a45da91

## Steps to QA

1. Go to http://localhost:3000/hacktoberfest/contributors/filter/recent?range=7
2. Switch the grid to card view by pressing the card view button on the grid.

CleanShot 2023-12-13 at 14 47 45

3. The grid loads up in card view.
4. Click 

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by OgDev-01** [comment]
This fixes the issue of clicking the button to expand and see pull requests, but i think we might need to revisit this UI-wise as the current behaviour when the button is clicked is kind of weird. cc: @isabensusan

**Review by nickytonline — COMMENTED** [review]
taking a peek at this to see if we can avoid z-index additions.

**Comment by nickytonline** [comment]
I took a peek at this @OgDev-01, and we do not need the z-index. The issue is the graph is overflowing. Here is the graph not overflowing showing the button working and as well, you can now click on the repository buttons. They didn't work before either.

CleanShot 2023-12-07 at 08 19 47

The light blue translucent box is the space that the graph takes up.

CleanShot 2023-12-07 at 08 21 51

The graph appears to be 404px x 300px. If we trim the bottom of the graph, we should be good. Currently looking how to do that as it's a third party component rendering a canvas.

**Comment by nickytonline** [comment]
I did some digging and we can use the style prop to modify the height. In the [docs](https://github.com/hustcc/echarts-for-react), it defaults to `300px`.

https://github.com/open-sauced/app/blob/6cc3acfa914ea8064ef4d02b78b4b5b00374b13d/components/atoms/EChartWrapper/echart-wrapper.tsx#L15-L25

I know we don't normally use explicit heights like this, but this is a canvas element that is being rendered.

```diff
 const EChartWrapper: React.FC<EChartWrapperProps> = ({ option, className }) => {
   return (
    <ReactECharts
       option={option}
       className={className || ""}
+      style={{ height: "145px" }}
       notMerge={true}
       lazyUpdate={true}
     />
   );
 };
```

**Comment by OgDev-01** [comment]
> I did some digging and we can use the style prop to modify the height. In the [docs](https://github.com/hustcc/echarts-for-react), it defaults to `300px`.
> 
> [`6cc3acf`/components/atoms/EChartWrapper/echart-wrapper.tsx#L15-L25](https://github.com/open-sauced/app/blob/6cc3acfa914ea8064ef4d02b78b4b5b00374b13d/components/atoms/EChartWrapper/echart-wrapper.tsx#L15-L25)
> 
> I know we don't normally use explicit heights like this, but this is a canvas element that is being rendered.
> 
> ```diff
>  const EChartWrapper: React.FC<EChartWrapperProps> = ({ option, className }) => {
>    return (
>     <ReactECharts
>        option={option}
>        className={className || ""}
> +      style={{ height: "145px" }}
>        notMerge={true}
>        lazyUpdate={true}
>      />
>    );
>  };
> ```
> 
> With these changes, we do not need a `z-index` and as well, you can now click on the repo buttons under the chart as well.

Thanks @nickytonline 👍, I'll apply this with a test to see if everything works fine as expected shortly

**Comment by OgDev-01** [comment]
Also checking to see if we are using this chart in other areas to avoid any breaking changes

**Comment by nickytonline** [comment]
@OgDev-01 is off, so getting this updated so we can get this fix out.

**Comment by nickytonline** [comment]
I reverted your last commit @OgDev-01 because removing the EChartWrapper is not a part of the fix. It's a refactor we can do later.

**Review by nickytonline — APPROVED** [review]

File: components/organisms/ContributorCard/contributor-card.tsx [hunk 1]
```diff
@@ -58,15 +58,15 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
           </div>
         </div>
         <div className="h-32">
-          <CardLineChart lineChartOption={chart} />
+          <CardLineChart lineChartOption={chart} height="140px" />
```
Comment (nickytonline) [inline components/organisms/ContributorCard/contributor-card.tsx:61]: `140px` seems to be the sweet spot for height as this item is a canvas.

CleanShot 2023-12-13 at 14 50 43

**Review by bdougie — APPROVED** [review]

**Pull request merged**